float64 me_dx
float64 me_dy
float64 me_vx
uint8 me_object_id
uint8 me_object_lane
float64 me_timestamp